### PR TITLE
feature/chuck-5-release  Chuck 5 release CHK5-330 

### PR DESCRIPTION
**Pull request by dnikitop**

### CODE REVIEW

File: amcl/test/basic_localization_stage.xml [hunk 1]
```diff
@@ -4,7 +4,7 @@ setting pose: 30.329 34.644 3.142
 <launch>
     <param name="/use_sim_time" value="true"/>
     <node name="rosbag" pkg="rosbag" type="play" 
-        args="-d 5 -r 1 --clock --hz 10 $(find amcl)/test/basic_localization_stage_indexed.bag"/>
+        args="-d 5 -r 5 --clock --hz 10 $(find amcl)/test/basic_localization_stage_indexed.bag"/>
```
Comment (DrCodeWizzard) [inline amcl/test/basic_localization_stage.xml:7]: is this change accelerating playback by 5x?

File: amcl/CMakeLists.txt [hunk 2]
```diff
@@ -108,24 +108,24 @@ if(CATKIN_ENABLE_TESTING)
...
   # Maps
   catkin_download_test_data(${PROJECT_NAME}_willow-full.pgm
     http://download.ros.org/data/amcl/willow-full.pgm
     DESTINATION ${CATKIN_DEVEL_PREFIX}/${CATKIN_PACKAGE_SHARE_DESTINATION}/test
     MD5 b84465cdbbfe3e2fb9eb4579e0bcaf0e)
-  catkin_download_test_data(${PROJECT_NAME}_willow-full-0.05.pgm
-    http://download.ros.org/data/amcl/willow-full-0.05.pgm
-    DESTINATION ${CATKIN_DEVEL_PREFIX}/${CATKIN_PACKAGE_SHARE_DESTINATION}/test
-    MD5 b61694296e08965096c5e78611fd9765)
+#  catkin_download_test_data(${PROJECT_NAME}_willow-full-0.05.pgm
```
Comment (DrCodeWizzard) [inline amcl/CMakeLists.txt:125]: why are we commenting out the tests

Comment (dnikitop) [reply amcl/CMakeLists.txt:125]: we aren't. This was just downloading data used for tests that we don't run anymore

File: amcl/test/basic_localization_stage.xml (hunk 1 above)
Comment (dnikitop) [reply amcl/test/basic_localization_stage.xml:7]: yes, it's making the test run faster.

**Review by DrCodeWizzard — APPROVED** [review]

**Pull request merged**